suite-native/graph/src/components/Graph.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { LineGraph } from 'react-native-graph';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box } from '@suite-native/atoms';
+
+import { TimeSwitch } from './TimeSwitch';
+
+const dummyData = [
```

Comment:
This should not be in this package, dummy data should be in place where this package is used, afterwards we will only change source if dummy data

Reply:
https://github.com/trezor/trezor-suite/pull/5838/commits/8d03ac13bbdc855f35c585b400951d9764b552cb